gulpfile.js

```diff
@@ -183,7 +183,6 @@ createTask('todos:find-closed', todosFindClosed);
 createTask('unit', unit);
 createTask('update-packages', updatePackages);
 createTask('validator', validator);
-createTask('validator-java', validatorJava);
```

Comment:
Likely need to remove the require for this value (`validatorJava`).

Reply:
Good catch. Done.